test/gtest/uct/ib/test_ib_event.cc

```diff
@@ -177,14 +260,29 @@ UCS_TEST_P(uct_p2p_test_event, last_wqe)
     }
 }
 
+ucs_log_level_t uct_p2p_test_event_log::orig_log_level;
+volatile unsigned uct_p2p_test_event_log::flushed_qp_num;
+
+UCT_INSTANTIATE_RC_TEST_CASE(uct_p2p_test_event_log);
+
+
+class uct_p2p_test_event : public uct_ep_test_event {
+public:
+    bool wait_for_last_wqe_event(entity& e, bool before) {
+        init_qp(e);
+        return uct_test_event_base::wait_for_last_wqe_event(e, before);
```

Comment:
how are we waiting for event without register?

Reply:
It's EP based test, EP init registers.

Reply:
so why is there "init_qp" before, if it's ep-based?

Reply:
to move common logic to uct_test_event_base which is agnostic of QP implementation

Reply:
/* retrieve rc qp from connected entity */